examples/basic/steady_state_thermal_analysis.py

```diff
@@ -0,0 +1,413 @@
+    Quantity("0 [sec]"),
+    Quantity("1 [sec]"),
+    Quantity("2 [sec]"),
+]
+RAD.AmbientTemperature.Output.DiscreteValues = [
+    Quantity("22[C]"),
+    Quantity("30[C]"),
+    Quantity("40[C]"),
+]
+RAD.Correlation = RadiationType.SurfaceToSurface
+
+ANLYS_SET = STAT_THERM.AnalysisSettings
+ANLYS_SET.NumberOfSteps = 2
+ANLYS_SET.CalculateVolumeEnergy = True
+
+STAT_THERM.Activate()
+ExtAPI.Graphics.Camera.SetFit()
+ExtAPI.Graphics.ExportImage(
+    os.path.join(cwd, "BoundaryConditions.png"), image_export_format, settings_720p
+)
+display_image("BoundaryConditions.png")
+
+# %%
+# Add Results
+STAT_THERM_SOLN = DataModel.Project.Model.Analyses[0].Solution
+TEMP_RST = STAT_THERM_SOLN.AddTemperature()
+TEMP_RST.By = SetDriverStyle.MaximumOverTime
+
+
+TEMP_RST2 = STAT_THERM_SOLN.AddTemperature()
+TEMP_RST2.Location = BODY1
+
+TEMP_RST3 = STAT_THERM_SOLN.AddTemperature()
+TEMP_RST3.Location = Path
+
+TEMP_RST4 = STAT_THERM_SOLN.AddTemperature()
+TEMP_RST4.Location = SURF
+
+
+# Total Heat Flux
+TOT_HFLUX = STAT_THERM_SOLN.AddTotalHeatFlux()
+DIR_HFLUX = STAT_THERM_SOLN.AddTotalHeatFlux()
+DIR_HFLUX.ThermalResultType = TotalOrDirectional.Directional
+DIR_HFLUX.NormalOrientation = NormalOrientationType.ZAxis
+
+LCS2.PrimaryAxisDefineBy = CoordinateSystemAlignmentType.GlobalZ
+DIR_HFLUX.CoordinateSystem = LCS2
+DIR_HFLUX.DisplayOption = ResultAveragingType.Averaged
+
+# Thermal Error
+THERM_ERROR = STAT_THERM_SOLN.AddThermalError()
+
+# Temperature Probe
+TEMP_PROBE = STAT_THERM_SOLN.AddTemperatureProbe()
+TEMP_PROBE.GeometryLocation = FACE1
+TEMP_PROBE.LocationMethod = LocationDefinitionMethod.CoordinateSystem
+TEMP_PROBE.CoordinateSystemSelection = LCS2
+
+# Heat Flux Probe
+HFLUX_PROBE = STAT_THERM_SOLN.AddHeatFluxProbe()
+HFLUX_PROBE.LocationMethod = LocationDefinitionMethod.CoordinateSystem
+HFLUX_PROBE.CoordinateSystemSelection = LCS2
+HFLUX_PROBE.ResultSelection = ProbeDisplayFilter.ZAxis
+
+
+# Reaction Probe
+ANLYS_SET.NodalForces = OutputControlsNodalForcesType.Yes
+REAC_PROBE = STAT_THERM_SOLN.AddReactionProbe()
+REAC_PROBE.LocationMethod = LocationDefinitionMethod.GeometrySelection
+REAC_PROBE.GeometryLocation = FACE1
+
+
+Rad_Probe = STAT_THERM_SOLN.AddRadiationProbe()
+Rad_Probe.BoundaryConditionSelection = RAD
+Rad_Probe.ResultSelection = ProbeDisplayFilter.All
+
+
+STAT_THERM_SOLN.Solve(True)
+
+# sphinx_gallery_start_ignore
+assert str(STAT_THERM_SOLN.Status) == "Done", "Solution status is not 'Done'"
+# sphinx_gallery_end_ignore
+
+
+# %%
+# All body temperature
+
+Tree.Activate([TEMP_RST])
+ExtAPI.Graphics.Camera.SetFit()
+ExtAPI.Graphics.ExportImage(
+    os.path.join(cwd, "temp.png"), image_export_format, settings_720p
+)
+display_image("temp.png")
+
+# %%
+# end of the body temperature
+
+Tree.Activate([TEMP_RST2])
+ExtAPI.Graphics.Camera.SetFit()
+ExtAPI.Graphics.ExportImage(
+    os.path.join(cwd, "temp2.png"), image_export_format, settings_720p
+)
+display_image("temp2.png")
+
+# %%
+# path
```

Comment:
Is this the path that the image is being exported to? I think it would be helpful to specify what path is